update-resources.sh

```diff
@@ -40,171 +40,173 @@ function chkfs() {
+  wget -q -O ./windows/SpoolSample.exe "https://gitlab.com/onemask/pentest-tools/-/raw/master/windows/SpoolSample.exe"
+  wget -q -O ./windows/SpoolSample_v4.5_x64.exe "https://gitlab.com/onemask/pentest-tools/-/raw/master/windows/SpoolSample_v4.5_x64..exe"
 }
 
 function add_diaghub() {
   info "Downloading DiagHub"
   chkfs "./windows/"
-  wget -nv -O ./windows/diaghub.exe "https://gitlab.com/onemask/pentest-tools/-/raw/master/windows/diaghub.exe"
+  wget -q -O ./windows/diaghub.exe "https://gitlab.com/onemask/pentest-tools/-/raw/master/windows/diaghub.exe"
 }
 
 function add_lazagne() {
   info "Downloading LaZagne"
-  git -C ./ clone https://github.com/AlessandroZ/LaZagne
+  git -C ./ clone -q https://github.com/AlessandroZ/LaZagne
   chkfs "./linux/"
+  rm -rf ./linux/LaZagne
   mv ./LaZagne/Linux ./linux/LaZagne
   chkfs "./mac/"
+  rm -rf ./mac/LaZagne
   mv ./LaZagne/Mac ./mac/LaZagne
   chkfs "./windows/"
+  rm -rf ./windows/LaZagne
   mv ./LaZagne/Windows ./windows/LaZagne
-  wget -nv -O ./windows/LaZagne/lazagne.exe "$(curl -s https://github.com/AlessandroZ/LaZagne/releases/latest | grep -o '"[^"]*"' | tr -d '"' | sed 's/tag/download/')/lazagne.exe"
+  wget -q -O ./windows/LaZagne/lazagne.exe "https://github.com/AlessandroZ/LaZagne/releases/latest/download/lazagne.exe"
   rm -rf ./LaZagne
 }
 
 function add_sublinacl() {
   info "Downloading Sublinacl"
   chkfs "./windows/"
-  wget -nv -O ./windows/sublinacl.exe "https://gitlab.com/onemask/pentest-tools/-/raw/master/windows/sublinacl.exe"
+  wget -q -O ./windows/sublinacl.exe "https://gitlab.com/onemask/pentest-tools/-/raw/master/windows/sublinacl.exe"
 }
 
 function add_plink() {
   info "Downloading plink"
   chkfs "./windows/"
-  wget -nv -O ./windows/plink32.exe "https://the.earth.li/~sgtatham/putty/latest/w32/plink.exe"
-  wget -nv -O ./windows/plink64.exe "https://the.earth.li/~sgtatham/putty/latest/w64/plink.exe"
+  wget -q -O ./windows/plink32.exe "https://the.earth.li/~sgtatham/putty/latest/w32/plink.exe"
+  wget -q -O ./windows/plink64.exe "https://the.earth.li/~sgtatham/putty/latest/w64/plink.exe"
 }
 
 function add_deepce() {
   info "Downloading deepce"
   chkfs "./linux/"
-  wget -nv -O ./linux/deepce.sh "https://github.com/stealthcopter/deepce/raw/master/deepce.sh"
+  wget -q -O ./linux/deepce.sh "https://github.com/stealthcopter/deepce/raw/master/deepce.sh"
 }
 
 function add_webshells() {
   info "Downloading webshells"
   chkfs "./webshells/PHP/wso-webshell/"
-  wget -nv -O ./webshells/PHP/wso-webshell/wso.php "https://raw.githubusercontent.com/mIcHyAmRaNe/wso-webshell/master/wso.php"
-  sed -i 's/fa769dac7a0a94ee47d8ebe021eaba9e/0fc3bcf177377d328c77b2b51b7f3c9b/g' ./webshells/PHP/wso-webshell/wso.php
+  wget -q -O ./webshells/PHP/wso-webshell/wso.php "https://raw.githubusercontent.com/mIcHyAmRaNe/wso-webshell/master/wso.php"
+  perl -i -pe's/fa769dac7a0a94ee47d8ebe021eaba9e/0fc3bcf177377d328c77b2b51b7f3c9b/g' ./webshells/PHP/wso-webshell/wso.php
```

Comment:
Let's use an OR operator here in order to have a fallback if perl/sed is not installed on the host: `command 1 || command 2`